static/app/views/explore/logs/index.tsx

```diff
@@ -90,6 +91,11 @@ export default function LogsPage() {
                   href={LOGS_INSTRUCTIONS_URL}
                   external
                   size="xs"
+                  onMouseDown={() => {
+                    trackAnalytics('logs.doc_link.clicked', {
```

Comment:
Shouldn't we add this to https://github.com/getsentry/sentry/blob/master/static/app/utils/analytics/logsAnalyticsEvent.tsx?